packages/shorebird_cli/lib/src/commands/patch_command.dart

```diff
@@ -269,7 +269,8 @@ Or downgrade your Flutter version and try again using:
   ${lightCyan.wrap('cd ${ShorebirdEnvironment.flutterDirectory.path}')}
   ${lightCyan.wrap('git checkout ${release.flutterRevision}')}
 
-${yellow.wrap('Warning: downgrading Flutter is not recommended and should only be used in emergencies (e.g. to patch a critical bug in production)')}.
+Eventually shorebird will support older flutter versions:
```

Comment:
Shorebird plans to support this automatically, let us know if it's important to you: